src/commands/src/helpers/dependency-helpers.ts

```diff
@@ -7,29 +7,28 @@ import { BuildDependencies } from '../../../entities/job';
 const existsAsync = promisify(fs.exists);
 const writeFileAsync = promisify(fs.writeFile);
 
-async function cloneRepo(repoName: string) {
-  await executeCliCommand({
-    command: 'git',
-    args: ['clone', `https://github.com/mongodb/${repoName}`],
-    options: { cwd: `${process.cwd()}/repos` },
-  });
-}
-async function createEnvProdFile(repoDir: string, projectName: string, baseUrl: string, prefix = '') {
+async function createEnvProdFile(
+  repoDir: string,
+  projectName: string,
+  baseUrl: string,
+  logger: (message: string) => void,
+  prefix = ''
+) {
   const prodFileName = `${process.cwd()}/snooty/.env.production`;
 
   try {
     await writeFileAsync(
       prodFileName,
       `GATSBY_SITE=${projectName}
       GATSBY_MANIFEST_PATH=${repoDir}/bundle.zip
-      GATSBY_PARSER_USER=${process.env.USER}
+      GATSBY_PARSER_USER=${process.env.USER ?? `docsworker-xlarge`}
```

Comment:
Super nit: Since we aren't doing any interpolation, we can use regular single quotes instead of backticks.